hash/spectrum_cass.xml

```diff
@@ -6119,16 +6114,16 @@ no loading screen, no custom loader
 	</software>
 
 	<software name="mortfilea" cloneof="cleversm">
-		<description>Mortadelo Y Filemon (Small Case) (Dro Soft)</description>
+		<description>Mortadelo y Filemon (Dro Soft, small case)</description>
 		<year>1988</year>
-		<publisher>Magic Bytes / Dro Soft</publisher>
+		<publisher>Dro Soft</publisher>
 		<part name="cass" interface="spectrum_cass">
 			<dataarea name="cass" size="50304">
 				<rom name="mortadelo y filemon - small case (dro soft).tzx" size="50304" crc="3079e2e2" sha1="4820dd28964c6fca6998822b8dcb47d14b9679c5"/>
 			</dataarea>
 		</part>
 	</software>
-
+	
```

Comment:
Stray whitespace here.  Please run srcclean on this file to reduce churn.

Reply:
Done it.